package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "studio-hiring",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
If you add a line here for starting the server up locally then you won't have to install `http-server` globally :+1:

Something like:
`"start": "http-server -p 9081"`

Then you can just run `npm start`

Reply:
nice suggestion - thank you.